src/app/shared/issue-pr-card/issue-pr-card.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-card class="card" [ngClass]="getIssueOpenOrCloseColorCSSClass()">
+  <a class="no-underline link-grey-dark" (click)="viewIssueInBrowser($event)">
+    <span [matTooltip]="this.issue.updated_at">
+      <mat-card-header>
+        <mat-card-title>
+          <span class="octicon" [octicon]="getOcticon()" [color]="getIssueOpenOrCloseColor()"></span>
+          #{{ issue.id }}: {{ fitTitleText() }}
+        </mat-card-title>
+      </mat-card-header>
```

Comment:
Let's refactor this into a separate component. We can pass the Octicon, Color, Id and the Title to this new component.